### ISSUE TITLE
Automatically set screen size incorrect for multiple monitors

### ISSUE THREAD

**Issue by cemulate** [jlam55555/veikk-linux-driver-gui#1]
In my case, I have 3 1080p monitors and 1 virtual screen, so my screen size should be `5760 x 1920`, but the program incorrectly assumes `1920 x 1080` and doesn't allow them to be changed/edited. After setting my `screen_size` and `screen_map` manually, the tablet works as expected, so `screen_size` really does need to be set to the total size of the virtual screen.

I can probably take a look at this if I get some time but just wanted to get the issue out there in case it's a quick fix.

**Comment by iamashwin99** [comment]
I have the same issue. I have 2 monitors side by side and I need the tablet to map only to the second monitor. However, I couldn't modify the screen* parameters.
My dual-screen setup is as follows.
![image](https://user-images.githubusercontent.com/46030335/80108617-1dd24a00-859a-11ea-9c47-b57e78ad3809.png)
and this is how I had to trick the config to work the way I intend it to work:
![image](https://user-images.githubusercontent.com/46030335/80108844-638f1280-859a-11ea-8ef2-7870e6b5df2b.png)
 so basically I had to take the "X map" and "map width" parameters to half the screen width. I hope this solution helps as a quick workaround

**Comment by jlam55555** [comment]
@iamashwin99 That's the current workaround for this kind of scenario, and unfortunately it's not documented. I had hoped to get to it much sooner but have gotten pretty busy. If I get the time to quickly look into Qt again, I'll try to fix it properly.
